documentation/adr/0002-wiki-platform.md

```diff
@@ -0,0 +1,294 @@
+# Communications Tooling: Wiki Platform
+
+- **Status:** Draft
+- **Last Modified:** 2023-06-15 <!-- REQUIRED -->
+- **Related Issue:** [#{issue number}](https://github.com/HHS/grants-api/issues) <!-- RECOMMENDED -->
+- **Deciders:** {list everyone involved in the decision} <!-- OPTIONAL -->
+- **Tags:** {space and/or comma separated list of tags} <!-- OPTIONAL -->
+
+## Context and Problem Statement
+
+The [communications platform milestone](milestone) identifies a series of platforms through which the Grants API project needs to engage both internal and external stakeholders. One of these platforms is a wiki for storing notes, documents, and other content about the project. Ideally we would select a platform that balances ease of use and flexibility with the cost of implementing and maintaining the wiki.
+
+The goal of this ADR is to evaluate a series of potential wiki platforms and determine which one best fits the needs and objectives of this project based on the decision criteria outlined below.
+
+## Decision Drivers <!-- RECOMMENDED -->
+
+#### Must Have
```

Comment:
Let's add AuthZ model per page: the ability to set some pages as viewable to the public, some as private to a group of users, etc. As well as the ability to control who can edit which content. We might want some pages that are publicly viewable but locked to editing by a small group.

Reply:
And let's add accessibility (for visually impaired and other users).

Also, available on restricted networks at HHS and, as a stretch goal, the White House (EOP).

Reply:
If we open a new PR just for `documentation/adr/0002-wiki-platform.md`, let's make sure it incorporates these comments?